src/main/kotlin/no/nav/syfo/application/api/authentication/JwtIssuer.kt

```diff
@@ -8,5 +8,6 @@ data class JwtIssuer(
 
 enum class JwtIssuerType {
     selvbetjening,
-    veileder
+    veileder,
+    VEILEDER_V2,
```

Comment:
Hvis dette bare har betydning internt i isdialogmote burde vi kanskje endre de to gamle typene til uppercase også?

Reply:
Ja, men tenker slike rydde-commits for warnings er bedre å gjøres separat.